From 50cd24ddb6f0bcae01623196515ce62527bb79df Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Thu, 4 Jan 2024 13:32:41 +0000
Subject: [PATCH] arm64: defconfig: remove CONFIG_IPQ_APSS_5018

This was added in 7f0c873 ("Enable IPQ5018 SoC base configs"), but there
is no IPQ_APSS_5018 config defined in the kconfig at present.

Signed-off-by: Ross Burton <ross.burton@arm.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 arch/arm64/configs/defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 929d436c612a..cb2abdb5d2d9 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -1205,7 +1205,6 @@ CONFIG_QCOM_CLK_SMD_RPM=y
 CONFIG_QCOM_CLK_RPMH=y
 CONFIG_IPQ_APSS_6018=y
 CONFIG_IPQ_GCC_5332=y
-CONFIG_IPQ_APSS_5018=y
 CONFIG_IPQ_GCC_5018=y
 CONFIG_IPQ_GCC_6018=y
 CONFIG_IPQ_GCC_8074=y
-- 
2.39.2

